projects/angular/src/popover/dropdown/_properties.dropdown.scss

```
@@ -5,50 +5,51 @@
  */
 @use '../../utils/mixins';
 @use '../../utils/variables/variables.global';
-@use 'variables.dropdown' as dropdown-variables;
+@use '@cds/core/tokens/tokens.scss';
 
 @include mixins.exports('dropdown.properties') {
-  @if variables.$clr-use-custom-properties == true {
-    :root {
+  :root {
+    [cds-theme] {
       // Usage: ../popover/dropdown/_dropdown.clarity.scss
-      --clr-dropdown-active-text-color: var(--clr-color-neutral-1000);
-      --clr-dropdown-active-background-color: var(--clr-global-selection-color);
-      --clr-dropdown-bg-color: var(--clr-color-neutral-0);
-      --clr-dropdown-border-color: var(--clr-color-neutral-400);
-      --clr-dropdown-divider-color: var(--clr-color-neutral-200);
-      --clr-dropdown-divider-border-width: var(--clr-global-borderwidth);
-      --clr-dropdown-divider-margin: #{variables.$clr_baselineRem_0_25 0};
-      --clr-dropdown-child-border-color: var(--clr-color-neutral-500);
-      --clr-dropdown-bg-hover-color: var(--clr-global-hover-color);
-      --clr-dropdown-selection-color: var(--clr-global-selection-color);
+      --clr-dropdown-active-text-color: #{tokens.$cds-alias-object-interaction-color-active};
+      --clr-dropdown-active-background-color: #{tokens.$cds-alias-object-interaction-background-active};
+      --clr-dropdown-bg-color: #{tokens.$cds-alias-object-overlay-background};
+      --clr-dropdown-border-color: #{tokens.$cds-alias-object-border-color};
+      --clr-dropdown-divider-color: #{tokens.$cds-alias-object-border-color-tint};
+      --clr-dropdown-divider-border-width: #{tokens.$cds-global-space-1};
+      --clr-dropdown-divider-margin: #{tokens.$cds-global-space-6};
+      --clr-dropdown-child-border-color: #{tokens.$cds-alias-object-border-color};
+      --clr-dropdown-bg-hover-color: #{tokens.$cds-alias-object-interaction-background-hover};
+      --clr-dropdown-selection-color: #{tokens.$cds-alias-object-interaction-background-selected};
       --clr-dropdown-box-shadow: var(--clr-popover-box-shadow-color);
-      --clr-dropdown-caret-icon-size: #{variables.$clr_baselineRem_0_5};
 
       // Usage:  ../data/datagrid/_datagrid.clarity.scss
       // Usage:  ../emphasis/alert/_alert.clarity.scss
-      --clr-dropdown-text-color: var(--clr-p1-color);
+      --clr-dropdown-text-color: var(--clr-global-font-color);
```

For what is this used?

```suggestion
      --clr-dropdown-text-color: tokens.$cds-alias-object-interaction-color;
```